.circleci/config.yml

```
@@ -267,7 +267,9 @@ commands:
     steps:
       - run:
           name: Install Python Dependencies with Pip
-          command: pip3 install --user -e .
+          command: |
+            python setup.py install_egg_info --install-dir=/home/circleci/.local/lib/python3.7/site-packages
```

```suggestion
            python setup.py install_egg_info --install-dir=/home/circleci/.local/lib/python3.7/site-packages
```
This is #1411 - should we be duplicating it here?

---

Right, we don't. It's just a placeholder before we merge #1411 to master. Historically, I tried this here before, and once it worked I created #1411, but I will remove this commit once that's merged.

---

✅